libs/blocks/chart/chartLightTheme.js

```diff
@@ -23,7 +23,7 @@ export default (deviceSize) => {
   const axisColor = '#767676';
   const options = {
     textStyle: {
-      fontFamily: 'Adobe Clean',
+      fontFamily: '"Adobe Clean", adobe-clean, "Trebuchet MS", sans-serif',
```

Comment:
Do we really need to set a font family here? Ideally we want to inherit from body, which has the font family defined.

Reply:
The chart library will add a font family if you don't set any so inheriting from body won't work.

Reply:
@meganthecoder - using `'inherit'` should do the trick
<img width="1568" alt="Screenshot 2025-02-20 at 17 02 56" src="https://github.com/user-attachments/assets/1a0fa6d7-f56d-40ba-a534-d6af2cd59545" />

Reply:
@narcis-radu Right, of course. Sorry, I didn't see this until after it got merged to stage. Do you want me to create a follow-up PR to change to `inherit`?

Reply:
@meganthecoder - only if it's not too much trouble. We don't change fonts that often, but if we do, it would be one less line to change if we use `inherit`. Thank you!

Reply:
PR https://github.com/adobecom/milo/pull/3742